### ISSUE TITLE
Donation amount is actually a multiplication factor

### ISSUE THREAD

**Issue by jorgesat** [YunoHost/pepettes#5]
Pepettes version: 1.0.1~ynh3
YNH version: 11.0.9.15 on VPS

The installed app is asking for an amount to be donated either recurrently or just one time.

However, when sent to the Stripe payment form, the amount given in the app becomes a multiplication factor of the set values in the Stripe product.

For example,

I have set up on Stripe to have a recurrent 1,50 EUR donation and a one-time donation of 10 EUR.

When choosing 10 EUR a month on Pepettes and clicking donate, Stripe will take it as 15 EUR per month (1,50 *  10).
When choosing 10 EUR one-time on Pepettes, Stripe will take it as a 100 EUR one-time donation (10 * 10).

**Comment by zamentur** [comment]
It's not a bug, it's the way to configure it, the idea is to define 1€ (or 1$) per product.

**Comment by jorgesat** [comment]
But the interface shows euros or dollars not amount of products.

**Comment by zamentur** [comment]
Indeed, but 10 x a donation product of 1€ make a donation of 10€...

If you want to make more complex things like a form to preselect specific amount or a form to buy object/services, you can just change the index.html (html form basis could be learn in day)
